src/cmds/versions/index.js

```diff
@@ -1,125 +1,129 @@
+    const output = [
+      `${chalk.bold('Version:')} ${version.version}`,
+      `${chalk.bold('Codename:')} ${version.codename || 'None'}`,
+      `${chalk.bold('Created on:')} ${version.createdAt}`,
+      `${chalk.bold('Released on:')} ${version.releaseDate}`,
+    ];
 
-  output.push(table.toString());
+    const table = new Table({
+      head: [chalk.bold('Is deprecated'), chalk.bold('Is hidden'), chalk.bold('Is beta'), chalk.bold('Is stable')],
+    });
 
-  return output.join('\n');
-};
+    table.push([
+      version.is_deprecated ? 'yes' : 'no',
+      version.is_hidden ? 'yes' : 'no',
+      version.is_beta ? 'yes' : 'no',
+      version.is_stable ? 'yes' : 'no',
+    ]);
 
-exports.run = function (opts) {
-  const { key, version, raw } = opts;
+    output.push(table.toString());
 
-  if (!key) {
-    return Promise.reject(new Error('No project API key provided. Please use `--key`.'));
+    return output.join('\n');
   }
 
-  const uri = version ? `${config.get('host')}/api/v1/version/${version}` : `${config.get('host')}/api/v1/version`;
-
-  return fetch(uri, {
-    method: 'get',
-    headers: cleanHeaders(key),
-  })
-    .then(handleRes)
-    .then(data => {
-      if (raw) {
-        return Promise.resolve(JSON.stringify(data, null, 2));
-      }
-
-      let versions = data;
-      if (!Array.isArray(data)) {
-        versions = [data];
-      }
-
-      if (!versions.length) {
-        return Promise.reject(
-          new Error(
-            `Sorry, you haven't created any versions yet! See \`${config.get('cli')} help ${
-              versionsCreate.command
-            }\` for commands on how to do that.`
-          )
-        );
-      }
-
-      if (version === undefined) {
-        return Promise.resolve(getVersionsAsTable(versions));
-      }
-
-      return Promise.resolve(getVersionFormatted(versions[0]));
-    });
+  async run(opts) {
+    const { key, version, raw } = opts;
+
+    if (!key) {
+      return Promise.reject(new Error('No project API key provided. Please use `--key`.'));
+    }
+
+    const uri = version ? `${config.get('host')}/api/v1/version/${version}` : `${config.get('host')}/api/v1/version`;
+
+    return fetch(uri, {
+      method: 'get',
+      headers: cleanHeaders(key),
+    })
+      .then(handleRes)
+      .then(data => {
+        if (raw) {
+          return Promise.resolve(JSON.stringify(data, null, 2));
+        }
+
+        let versions = data;
+        if (!Array.isArray(data)) {
+          versions = [data];
+        }
+
+        if (!versions.length) {
+          return Promise.reject(
+            new Error(
+              `Sorry, you haven't created any versions yet! See \`${config.get('cli')} help ${
+                new CreateVersionCmd().command
+              }\` for commands on how to do that.`
+            )
+          );
+        }
+
+        if (version === undefined) {
+          return Promise.resolve(VersionsCommand.getVersionsAsTable(versions));
```

Comment:
Since `getVersionsAsTable` is a static method we need to call it off the main class, `this.getVersionsAsTable` won't work. The method doesn't really need to be static, but since it doesn't use any internal command data it probably makes sense for it to be static?

If not, we'll need to alias `this` to `self` at the top of `run` to get around scope issues within the `fetch` thenable that this line is at.